### PR TITLE
SmthJedi update

### PR DESCRIPTION
**Pull request by NoSloppy**
Hierarchy made to be FAT32 friendly.
Added boot.wav.
Added config.ini.
Edited smoothsw.ini.
Normalized hum to -5db
High passed hum @100Hz - 48db slope.
Edited lock to have less monophonic-like hum mixed in.

### CODE REVIEW

File: SmthJedi/license.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
```
Comment (profezzorn) [inline SmthJedi/license.txt:2]: Should probably add that the original files are made by Thexter.

Comment (NoSloppy) [reply SmthJedi/license.txt:2]: added to all 4 fonts.

**Comment by profezzorn** [comment]
Why is hum called "hum01.wav" instead of just "hum.wav"?
No need for numbers if there is only one file.

File: SmthJedi/config.ini [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ProffieOSClashImageDuration=500.0
+
+# For OLED displays, the time a force.bmp will play
+ProffieOSForceImageDuration=1000.0
+
+### ---- These below as of OS6 only ---- ###
+
+# Minimum acceleration for Accent Swing file Selection
+#recommended value is 20.0 ~ 30.0
+ProffieOSMinSwingAcceleration=20.0
```
Comment (profezzorn) [inline SmthJedi/config.ini:70]: Are swings in progressive order?

File: SmthJedi/license.txt [hunk 3]
```diff
@@ -1 +1,3 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+Smoothswing sound font for ProffieOS by Thexter.
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
+boot.wav made by Brian Conner using a few with sounds sourced from creative-commons licensed freesound.org.
```
Comment (profezzorn) [inline SmthJedi/license.txt:3]: Please list all modifications you made.

File: SmthJedi/smoothsw.ini [hunk 4]
```diff
@@ -1,16 +1,46 @@
...
+# be played when we reach this swing speed.
+# Unit is degrees per second, 450 is a reasonable value.
+AccentSwingSpeedThreshold=450
+
+# If not zero AND accent swings are on, this defines the threshold for when
+# a swing is considered a slash. Unit is degrees per second **per second**.
+# NOTE - While 260 is the default value, it is subjective.
+# Something like 100 might work better.
+# The higher the accent swing threshold, the higher the slash threshold will need to be.
+AccentSlashAccelerationThreshold=260.0
```
Comment (profezzorn) [inline SmthJedi/smoothsw.ini:46]: This might be confusing if there are no slash effects present.

**Pull request closed without merging**